test/byContract/IexecEscrow/IexecEscrowToken.js.skip

```diff
@@ -1,903 +0,0 @@
-// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
```

Comment:
Maybe you can add "[Should not deposit/Should not deposit for] if not approved"

Reply:
[add test revert not approved spending](https://github.com/iExecBlockchainComputing/PoCo/pull/143/commits/6a2d4c41fba6c1e44781020d4f3236ca7662e210)